### PR TITLE
created home page UI

### PR DESCRIPTION
**Pull request by anshikaa12**
>> created a simple UI for home page
image

### CODE REVIEW

**Review by DeshmukhAkash2000 — APPROVED** [review]
everything looks good to me. good to go.

File: package.json [hunk 1]
```diff
@@ -34,5 +34,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-router-dom": "^6.3.0"
```
Comment (biswanath-tewari) [inline package.json:39]: router lib should not come under dev dependency

File: src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Home from "./home/home";
```
Comment (deepak29-git) [inline src/pages/index.js:1]: this is good approach to make index file and export all files from here

**Review by deepak29-git — APPROVED** [review]
Liked: your folder structure is good and code is very clean

**Review by biswanath-tewari — APPROVED** [review]
The code is quite clean.

**Pull request merged**